Fatras/include/ActsFatras/Digitization/ChannelMerger.hpp

```diff
@@ -0,0 +1,65 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Surfaces/Surface.hpp"
+#include "Acts/Surfaces/SurfaceError.hpp"
+#include "Acts/Utilities/Helpers.hpp"
+#include "ActsFatras/Digitization/DigitizationData.hpp"
+
+#include <array>
+#include <utility>
+
+namespace ActsFatras {
+
+/// Generic implementation of a channel merger, currently only additive
+/// channel merging
```

Comment:
```suggestion
/// Generic implementation of a channel merger, currently only additive
/// channel merging.
```
Doxygen autobrief requires a punctuation mark, I think.